### DISCUSSION TITLE
Invalid hook call

### DISCUSSION THREAD

**Discussion by drzax** [zeit-ui/react#482] [Q&A]
I feel like I must be missing something really obvious here, but I can't seem to use `<GeistProvider>` at all.

As soon as I add it to even the most basic boilerplate app, I get an `Uncaught Error: Invalid hook call.` error.

```js
render(
    <GeistProvider>
      <CssBaseline />
      <App {...appProps} />
    </GeistProvider>,
    appMountEl
  );
```

The `<App {...appProps} />` here is a more or less empty boilerplate that works perfectly until it's wrapped in `<GeistProvider>`. 

`The above error occurred in the <GeistProvider> component: GeistProvider@https://localhost:8000/index.js:6902:16`

Versions:
```
"@geist-ui/react": "^2.1.3",
"react": "^17.0.1",
"react-dom": "^17.0.1"
```

**Comment by unix** [comment]
Here is a simple [online example](https://codesandbox.io/s/dazzling-sound-jm23w?file=/src/index.js) for your reference, 
If you are still experiencing this issue, please provide an **online example** that will make it easier for me to help you.
